strategies/BidAskSpreadMA/consolidating-MA-buyside-wmaticuusdt.rain

```diff
@@ -159,16 +165,16 @@ moving-average-length: price-average-time,
 price-average: uniswap-v3-twap-output-ratio(
     buy-token sellback-token 
     price-average-time 0
-    poolfee
+    uniswap-v3-factory uniswap-v3-init-code poolfee
```

Comment:
updated uniswap parser needs factory and init-code literal can we have the same for rest of them?